### DISCUSSION TITLE
Looking for React Developers

### DISCUSSION THREAD

**Discussion by codyhandshaw** [reactplay/react-play#146] [Jobs]
Hello all, I am looking for React Engineers to join my team at LogicMonitor. Please let me know if you, or anyone else is interested please reach out to me!

Thanks,
Cody

**Comment by atapas** [comment]
Wow that is great @codyhandshaw 🔥. Could you please give more details about it?

- Is it a remote position?
- What sort of React experience and total industry experience you are looking for?

Thanks a lot for posting about it here.

**Comment by Shivam-Katare** [comment]
> Hello all, I am looking for React Engineers to join my team at LogicMonitor. Please let me know if you or anyone else is interested please reach out to me!
> 
> Thanks, Cody

Hey  @codyhandshaw could you give us more details? Like: Experience, or position.

**Comment by Angryman18** [comment]
> Hello all, I am looking for React Engineers to join my team at LogicMonitor. Please let me know if you, or anyone else is interested please reach out to me!
> 
> Thanks, Cody

i would like know some further info and requirement details.

**Comment by hsouiti** [comment]
> Hello all, I am looking for React Engineers to join my team at LogicMonitor. Please let me know if you, or anyone else is interested please reach out to me!
> 
> Thanks, Cody

I am interested. can you provide more infos.
thanks

**Comment by vasantisuthar** [comment]
> Hello all, I am looking for React Engineers to join my team at LogicMonitor. Please let me know if you, or anyone else is interested please reach out to me!
> 
> Thanks, Cody

I'm interested, would like to know more.

**Comment by codyhandshaw** [comment]
Apologies on the delay, I am a new GitHub user. I am looking for 3+ years in JavaScript specifically with React. Please email me your resume at cody.handshaw@logicmonitor.com or apply on our site here. (https://www.logicmonitor.com/career/?gh_jid=3742689)

**Reply by atapas** [reply]
Thanks, @codyhandshaw for the response.

**Reply by atapas** [reply]
@Angryman18 @vasantisuthar @hsouiti @Shivam-Katare

Hope you saw the above response. Tagging you as you showed interest in it.

**Reply by Shivam-Katare** [reply]
Yes! I responded.

**Reply by Angryman18** [reply]
i had started learning React 1 year ago but its the 3+ years requirements so i didnt respond actually.
